introduction.md

```diff
@@ -0,0 +1 @@
+Hello Everyone , Myself Hardik Charadva ,Currently Working in Orbit Tech
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.